### ISSUE TITLE
[Feature Request] Save Individual 'Styles' For Individual Subreddits 

### ISSUE THREAD

**Issue by sliceanddice9** [joelacus/RedditEnhancer#23]
The default styles of certain subreddits are quite unappealing, or even so poorly designed with the selection of text, font, boarder, background and colours that they are hard to read and navigate. 

The highly customizable 'Style' setting within the Reddit Enhancer add-on helps with correcting some of these subreddit UI elements to something that is much more functional and appealing to the eye.

If the user could configure a 'set and forget' saved customized style that **ONLY** applies to individual subreddits then that would be great.

**Comment by joelacus** [comment]
I have been thinking recently about subreddit specific options actually. I could add a text input area where the user can list subreddits, with an option next to it to either use the styles globally except these subreddits, or, disable globally and only show on these subreddits.

Btw, if you don't want to see any community themes, you can disable Community Themes in the Feed Settings in the User Settings. [https://www.reddit.com/settings/feed](https://www.reddit.com/settings/feed)

**Comment by joelacus** [comment]
Hi. I've just published version 0.18.0 with this feature. It will update automatically soon.

**Comment by sliceanddice9** [comment]
Thanks for the update.

Not sure if I'm doing something wrong but I've tried adding: **r/revancedapp** to the whitelist for my theme colour customization of the subreddit but the changes don't seem to stick. Whenever I refresh the page or if I close and then re-open  the subreddit; the style seems to reset back to its default form.

**Comment by joelacus** [comment]
Did you add the "r/"? The list only requires the sub name "revancedapp" and then use a comma to add another sub if you want. I'll make an update soon to handle this better so it doesn't get confused.

**Comment by sliceanddice9** [comment]
> Did you add the "r/"? The list only requires the sub name "revancedapp" and then use a comma to add another sub if you want. I'll make an update soon to handle this better so it doesn't get confused.

Yes. I recorded a video of the problem I'm having with the theme colour customization I make that is individually only applied for the **r/revancedapp** subreddit. 

As you can see in the video - after refreshing the browser window by clicking on the refresh icon - the custom style I created for **r/revancedapp**  is now gone and it resets back to the default style.  




https://github.com/joelacus/RedditEnhancer/assets/59092472/fdd98b3d-78ac-4a8a-832c-b9f9aaf68c77

**Comment by joelacus** [comment]
Sorry for the confusion. Remove the "r/" from the whitelist, it only requires the subreddit name. I'll work on an update so it'll work with the "r/" or not.

**Comment by sliceanddice9** [comment]
> Sorry for the confusion. Remove the "r/" from the whitelist, it only requires the subreddit name. I'll work on an update so it'll work with the "r/" or not.

Unfortunately it's still NOT working. The same thing happens. After refreshing the page - the custom white style I made of the subreddit reverts back to its default black style. Here is another video I recorded with the subredit name change to:  **"revancedapp"**


https://github.com/joelacus/RedditEnhancer/assets/59092472/ece97d34-cf19-47b7-adfa-139d1a3efc83
